### PR TITLE
Update ASE template with API, new management IPs

### PR DESCRIPTION
**Pull request by jftl6y**
### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use a parameter for resource locations with the defaultValue set to resourceGroup().location
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values)
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### Changelog

*
*
*

### CODE REVIEW

File: App-Service-Environment-AzFirewall/nestedtemplates/vnet.json [hunk 1]
```diff
@@ -108,24 +108,27 @@
...
                             "sourcePortRange": "*",
-                            "destinationPortRange": "454-455",
+                            "destinationPortRange": "*",
+                            "sourceAddressPrefix": "AppServiceManagement",
                             "destinationAddressPrefix": "*",
                             "access": "Allow",
                             "priority": 100,
                             "direction": "Inbound",
-                            "sourceAddressPrefixes": "[parameters('aseManagementIps')]"
+                            "sourcePortRanges": [],
```
Comment (bmoore-msft) [inline App-Service-Environment-AzFirewall/nestedtemplates/vnet.json]: remove empty properties

File: App-Service-Environment-AzFirewall/azuredeploy.json [hunk 2]
```diff
@@ -57,7 +57,7 @@
         },
         "_artifactsLocation": {
             "type": "string",
-            "defaultValue": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/App-Service-Environment-AzFirewall/",
+            "defaultValue": "https://raw.githubusercontent.com/azure/azure-quickstart-templates/master/App-Service-Environment-AzFirewall/",
```
Comment (bmoore-msft) [inline App-Service-Environment-AzFirewall/azuredeploy.json]: revert this change - Azure needs to be capitalized

Comment (jftl6y) [reply App-Service-Environment-AzFirewall/azuredeploy.json]: Fixed

File: App-Service-Environment-AzFirewall/nestedtemplates/vnet.json (hunk 1 above)
Comment (jftl6y) [reply App-Service-Environment-AzFirewall/nestedtemplates/vnet.json]: Fixed
